packages/fxa-auth-server/lib/routes/password.ts

```diff
@@ -1011,14 +1013,20 @@ module.exports = function (
           ),
         ]);
 
+        // Users that have 2FA enabled must verify the passwordForgotToken with
+        // authenticator code or recovery code to be granted resetToken
+        // const hasTotp = await otpUtils.hasTotpToken(passwordForgotToken);
+        // if (hasTotp && passwordForgotToken.verificationMethod <= 1) {
+        //   throw error.unverifiedSession();
+        // }
```

Comment:
Did you mean to leave this commented out?

Reply:
Whoops, meant to remove this. Because recovery key is considered 2FA I had to move this into the `/account/reset` route.